makefiles/tools/gdb.inc.mk

```
@@ -1,7 +1,11 @@
-# new versions of gdb will support all architectures in one binary
-ifeq ($(shell gdb-multiarch -v > /dev/null 2>&1; echo $$?),0)
-export GDB        ?= gdb-multiarch
+export GDBPREFIX    ?= $(PREFIX)
```

There is no reason to use additional spaces here.
```suggestion
export GDBPREFIX ?= $(PREFIX)
```